lib/backend/dynamo/configure.go

```diff
@@ -58,22 +59,30 @@ type AutoScalingParams struct {
 }
 
 // SetAutoScaling enables auto-scaling for the specified table with given configuration.
-func SetAutoScaling(ctx context.Context, svc *applicationautoscaling.ApplicationAutoScaling, tableName string, params AutoScalingParams) error {
+func SetAutoScaling(ctx context.Context, svc *applicationautoscaling.ApplicationAutoScaling, resourceID string, params AutoScalingParams) error {
+	readDimension := applicationautoscaling.ScalableDimensionDynamodbTableReadCapacityUnits
+	writeDimension := applicationautoscaling.ScalableDimensionDynamodbTableWriteCapacityUnits
+
+	if strings.Contains(resourceID, "/index/") {
```

Comment:
I would either add a comment here or move this into a helper function `isTableIndex` or something, otherwise this check looks pretty puzzling at first.

Reply:
I'll add a comment